snippets/properties.json

```diff
@@ -0,0 +1,13 @@
+{
+	"Add datasource properties": {
+		"prefix": "qds",
+		"body": [
+			"quarkus.datasource.url=${1:jdbc:mariadb://localhost:3306/mydb}",
```

Comment:
Why not using choice https://code.visualstudio.com/docs/editor/userdefinedsnippets#_choice to provide a list of syntax URL (according JDBC driver)?